### PR TITLE
FIRECERT-2199: Fix for testcase failing at validation for Lifecycle history

### PR DESCRIPTION
**Pull request by SnehalP251**

### CODE REVIEW

File: cypress/support/cypress-commands/commands.js [hunk 1]
```diff
@@ -699,7 +699,11 @@ Cypress.Commands.add('launchApp', (appType, appCallSign) => {
         },
       },
     };
-    requestMap.params.intent.data = data;
+    const messageIntent = {
+      action: CONSTANTS.SEARCH,
+      data: data,
+    };
```
Comment (neeradanelxsi) [inline cypress/support/cypress-commands/commands.js:705]: do we really need to create messageIntent twice ? https://github.com/rdkcentral/firebolt-certification-suite/pull/138/files#diff-a0c4b6497a6be8ded74b56b3268804337f0607a41306ae7c236b91b7a7fab118R678
Cant we just create the data object before and then at the last create a single messageIntent

Comment (SnehalP251) [reply cypress/support/cypress-commands/commands.js]: Addressed above review comment

File: cypress/support/cypress-commands/commands.js [hunk 2]
```diff
@@ -658,9 +658,6 @@ Cypress.Commands.add('launchApp', (appType, appCallSign) => {
       ? UTILS.getEnvVariable(CONSTANTS.THIRD_PARTY_APP_ID)
       : UTILS.checkForSecondaryAppId(appCallSign); // this is for the app to know the appId used for launch, so that it can use the same for creating PubSub connection.
   // if appType is certification, the appLaunch is for certification purposes. In such a case, discovery.launch should go with a basic intent that has the appId and the certification app role.
-  // create the request map
-  // basic intent to be sent to the app on launch
```
Comment (neeradanelxsi) [inline cypress/support/cypress-commands/commands.js:662]: Add comment here 
// Creating data for basic intent to be sent to the app on launch

File: cypress/support/cypress-commands/commands.js [hunk 3]
```diff
@@ -675,16 +672,6 @@ Cypress.Commands.add('launchApp', (appType, appCallSign) => {
...
-    const messageIntent = {
-      action: CONSTANTS.SEARCH,
-      data: data,
-      context: { source: CONSTANTS.DEVICE },
-    };
-
-    requestMap = {
-      method: CONSTANTS.DISCOVERY_LAUNCH,
-      params: { [CONSTANTS.APP_ID]: appId, [CONSTANTS.INTENT]: messageIntent },
-    };
```
Comment (neeradanelxsi) [inline cypress/support/cypress-commands/commands.js:687]: Add comment before the if condition. 
// If testType == lifecycle, modifying data to include _lifecycle_validation = true_ in the intent to be sent to the app

File: cypress/support/cypress-commands/commands.js [hunk 4]
```diff
@@ -699,8 +686,18 @@ Cypress.Commands.add('launchApp', (appType, appCallSign) => {
         },
       },
     };
-    requestMap.params.intent.data = data;
```
Comment (neeradanelxsi) [inline cypress/support/cypress-commands/commands.js:702]: Remove comments from line 695 and 696. 
Add comment before creating the messageIntent object
// Creating intent and request map to be sent to the app on launch

**Review by neeradanelxsi — DISMISSED** [review]
Added minor comments

**Review by jbigel — APPROVED** [review]

File: cypress/support/cypress-commands/commands.js (hunk 2 above)
Comment (SnehalP251) [reply cypress/support/cypress-commands/commands.js:662]: Added comment

File: cypress/support/cypress-commands/commands.js (hunk 3 above)
Comment (SnehalP251) [reply cypress/support/cypress-commands/commands.js:687]: Added comment

File: cypress/support/cypress-commands/commands.js (hunk 4 above)
Comment (SnehalP251) [reply cypress/support/cypress-commands/commands.js:702]: Added comment

**Review by neeradanelxsi — APPROVED** [review]

**Pull request merged**